diff --git a/content/browser/service_worker/service_worker_content_settings_proxy_impl.h b/content/browser/service_worker/service_worker_content_settings_proxy_impl.h
index 49a04df206daaf3a1bb7a12b0e79497c26210c39..d58b043feec8556988442a9ab9c48a05582278a3 100644
--- a/content/browser/service_worker/service_worker_content_settings_proxy_impl.h
+++ b/content/browser/service_worker/service_worker_content_settings_proxy_impl.h
@@ -40,6 +40,8 @@ class ServiceWorkerContentSettingsProxyImpl final
   void AllowWebLocks(AllowCacheStorageCallback callback) override;
   void RequestFileSystemAccessSync(
       RequestFileSystemAccessSyncCallback callback) override;
+  void AllowFingerprinting(AllowFingerprintingCallback callback) override;
+  void GetBraveFarblingLevel(GetBraveFarblingLevelCallback callback) override;
 
  private:
   const url::Origin origin_;
