diff --git a/content/public/browser/content_browser_client.cc b/content/public/browser/content_browser_client.cc
index 248a90c7682001296d623eb80e925f03360a5510..b48240ab48a9d73141f3708e26e8d289191f570e 100644
--- a/content/public/browser/content_browser_client.cc
+++ b/content/public/browser/content_browser_client.cc
@@ -447,6 +447,20 @@ bool ContentBrowserClient::AllowWorkerWebLocks(
   return true;
 }
 
+bool ContentBrowserClient::AllowWorkerFingerprinting(
+    const GURL& url,
+    BrowserContext* browser_context,
+    const std::vector<GlobalRenderFrameHostId>& render_frames) {
+  return true;
+}
+
+uint8_t ContentBrowserClient::WorkerGetBraveFarblingLevel(
+    const GURL& url,
+    BrowserContext* browser_context,
+    const std::vector<GlobalRenderFrameHostId>& render_frames) {
+  return 1 /* OFF */;
+}
+
 ContentBrowserClient::AllowWebBluetoothResult
 ContentBrowserClient::AllowWebBluetooth(
     content::BrowserContext* browser_context,
